main.tf

```diff
@@ -362,6 +363,26 @@ module "vm_mig" {
     }] : []
   )
   target_size = var.node_count
+  stateful_ips = var.enable_ssh ? [
+    {
+      interface_name = "nic0"
+      delete_rule    = "ON_PERMANENT_INSTANCE_DELETION"
+      is_external    = true
+    }
+  ] : []
+  update_policy = var.enable_ssh ? [
+    {
+      type                         = "OPPORTUNISTIC"
+      instance_redistribution_type = "NONE"
+      minimal_action               = var.is_replicated_deployment ? "RESTART" : "REPLACE"
+      max_unavailable_fixed        = var.is_replicated_deployment ? 3 : null
+      max_surge_fixed              = null
+      max_surge_percent            = null
+      max_unavailable_percent      = null
+      min_ready_sec                = null
+      replacement_method           = null
+    }
+  ] : []
```

Comment:
Would it make sense to add a comment here, explaining in how far these values are related to `enable_ssh`?

Reply:
The PR title and the summary do not mention `enable_ssh` at all. Is that intentional? 
There also is no ticket linked. 

Could you please extend the description and / or link a ticket with context?

Reply:
Changes added